src/encounter/index.ts

```diff
@@ -208,9 +211,14 @@ export class EncounterParser {
             hp: number,
             ac: number | string,
             mod: number,
-            xp: number;
+            xp: number,
+            friendly: boolean = false;
 
         if (typeof monster == "string") {
+            if (monster.match(/,\s+friend/) || monster.match(/,\s+ally/)) {
```

Comment:
Should you add the comma and space in the match? Is there a situation where that won’t be there?

Reply:
1. I meant to use `*` and not `+` .. so I'll fix that.
2. This is before the split, and the name is always first. I was worried about the deviant case where someone uses ally in the name.. comma ensures it is not found unless it is its own segment. 
  ```
  name = monster.split(/,\s?/)[0]; 
  ```